packages/adblocker/src/filters/network.ts

```diff
@@ -1275,16 +1452,39 @@ export default class NetworkFilter implements IFilter {
     return this.getMask() & FROM_ANY;
   }
 
+  private getOptionValue() {
+    return this.optionValue || '';
```

Comment:
I would keep it undefined if there isn't one instead of returning an empty string.

Reply:
This property is now dropped as `optionValue` is public property of `NetworkFilter` (which means we don't need this method). This discussion should be moved to `public getRedirect` method of `NetworkFilter`.